lib/ldp.js

```diff
@@ -229,8 +229,10 @@ class LDP {
 
     // First check if we are above quota
     let isOverQuota
+    // For some reason someone thought it was a bright idea to make url actually a
+    // req sometimes but not all the time 🙄. So Now I have to account for that
```

Comment:
Can you specify who "I" is, or just rewrite this so it's more "us" and "we" or "the code needs to ..."